draft-ounsworth-cfrg-kem-combiners.mkd

```diff
@@ -321,6 +371,17 @@ More precisely, for a given capacity `c` the indifferentiability proof shows tha
 For a random oracle, a difference in only a single bit gives an unrelated, uniformly random output.
 Hence, to be able to distinguish a key `k`, derived from shared keys `k_i` from a random bit string, an adversary has to correctly guess all key shares `k_i` entirely.
 
+The proposed construction in {{sec-kem-combiner}} with the instantiations in
+{{sec-instantiation}} preserves IND-CCA2 of any of its ingredient KEMs, i.e.
+the newly formed combined KEM is IND-CCA2 secure as long as at least one of the
+ingredient KEMs is. Indeed, the above stated indifferentiability from a random
+oracle qualifies Keccak as a split-key pseudorandom function as defined in
+{{KEMCombiners}}. That is, Keccak behaves like a random function if at least
+one input shared secret `ss_i` is picked uniformly at random. Our construction
+can thus be seen as an instantiation of the IND-CCA2 preserving Example 3 in
+Figure 1 of {{KEMCombiners}}, up to some reordering of input shared secrets and
+ciphertexts and their potential compression by a cryptographic hash function. 
```

Comment:
I wrote "potential" here because of the previously present options k_i = ss_i || ct_i or k_i = H(ss_i || ct_i). That word should be removed (in the future) if only the option k_i = H(ss_i || ct_i) persists.